pyproject.toml

```diff
@@ -35,6 +35,8 @@ google-api-python-client = "^2.90.0"
 chispa = "^0.9.2"
 grpcio = ">=1.48.1"
 plotly = "^5.15.0"
+pyarrow = "^12.0.1"
```

Comment:
Let's make it consistent with Spark:
https://github.com/apache/spark/blob/master/python/setup.py#L134
pyarrow = ">=4.0.0"